janusgraph-core/src/main/java/org/janusgraph/diskstorage/common/DistributedStoreManager.java

```diff
@@ -138,25 +138,25 @@ public String toString() {
     /**
      * This method attempts to generate Rid in the following three ways, in order,
      * returning the value produced by the first successful attempt in the sequence.
-     * <p/>
+     * <p>
      * <ol>
      * <li>
-     * If {@code config} contains {@see GraphDatabaseConfiguration#INSTANCE_RID_RAW_KEY},
+     * If {@code config} contains {@link GraphDatabaseConfiguration#INSTANCE_RID_RAW_KEY},
      * then read it as a String value.  Convert the String returned into a char[] and
      * call {@code org.apache.commons.codec.binary.Hex#decodeHex on the char[]}.  The
      * byte[] returned by {@code decodeHex} is then returned as Rid.
      * </li>
      * <li>
-     * If {@code config} contains {@see GraphDatabaseConfiguration#INSTANCE_RID_SHORT_KEY},
-     * then read it as a short value.  Call {@see java.net.InetAddress#getLocalHost()},
-     * and on its return value call {@see java.net.InetAddress#getAddress()} to retrieve
+     * If {@code config} contains {@link GraphDatabaseConfiguration#INSTANCE_RID_SHORT_KEY},
+     * then read it as a short value.  Call {@link java.net.InetAddress#getLocalHost()},
+     * and on its return value call {@link java.net.InetAddress#getAddress()} to retrieve
      * the machine's IP address in byte[] form.  The returned Rid is a byte[] containing
      * the localhost address bytes in its lower indices and the short value in its
      * penultimate and final indices.
      * </li>
      * <li>
      * If both of the previous failed, then call
-     * {@see java.lang.management.RuntimeMXBean#getName()} and then call
+     * {@link java.lang.management.RuntimeMXBean#getName()} and then call
      * {@code String#getBytes()} on the returned value.  Return a Rid as described in the
      * previous point, replacing the short value with the byte[] representing the JVM name.
      * </li>
```

Comment:
Not all of the `@link` elements here end up resolving. However, this entire javadoc block is actually for the method `getRid()` which is entirely commented out. In the produced HTML file, this javadoc is getting associated incorrectly with the `sleepAfterWrite()` method. The commented out method `getRid()` and the javadoc for it should be removed. It was commented out [4 years ago](https://github.com/JanusGraph/janusgraph/blame/master/janusgraph-core/src/main/java/org/janusgraph/diskstorage/common/DistributedStoreManager.java#L168), long before the JanusGraph fork.